src/readwrite.rs

```diff
@@ -114,7 +114,7 @@ impl<K> Store<K> where K: AsRef<[u8]> {
         })
     }
 
-    pub fn write<'env>(&mut self, env: &'env Rkv) -> Result<Writer<'env, K>, lmdb::Error> {
+    pub fn write<'env>(&self, env: &'env Rkv) -> Result<Writer<'env, K>, lmdb::Error> {
```

Comment:
It’s worth documenting right here that this will block if any other writers exist for this store.

Reply:
Good idea, noted in 39d7e5d.